server/src/main/java/io/spine/server/model/ExternalCommandReceiverMethodError.java

```diff
@@ -49,11 +50,15 @@
 
     private static final long serialVersionUID = 0L;
 
+    /**
+     * Creates a new exception for the command handler that violates the {@code @Command} semantic.
+     */
+    @Internal
     public ExternalCommandReceiverMethodError(
             CommandHandlingClass<?, ?> classWithViolation,
             Collection<? extends CommandAcceptingMethod<?, ?>> invalidMethods) {
-        super("The class `%s` declares `external` command receiver methods for command types: %s."
-                      + " Only event accepting methods should be marked as `external`.",
+        super("The class `%s` declares `external` command receiver methods for command types: %s. " +
```

Comment:
Please move `+` to next line. We do not check it automatically yet. But soon we'd do using Checkstyle.